packages/smooth_app/lib/pages/product/summary_card.dart

```diff
@@ -464,7 +464,7 @@ class _SummaryCardState extends State<SummaryCard> {
               },
               child: SmoothCard(
                 margin: EdgeInsets.zero,
-                color: const Color(0xfff5f6fa),
+                color: Theme.of(context).primaryColor,
```

Comment:
I think we rather use `Theme.of(context).colorScheme.primary` in the rest of the app, could you double-check that?

Reply:
Sure, let me fix that

Reply:
@GowthamGoush I have no opinion regarding colors, only about code: I don't like the possible confusion. If I have a look at the current code I only see `colorScheme.primary` and never `Theme.of(context).primaryColor`. So let's definitely use `colorScheme.primary` (and use its value if necessary).

Reply:
Yes, I definitely agree with whatever you mentioned regarding the standards that we follow. I will fix that soon. Also, thanks for the kind suggestion and feedback.

Reply:
I totally agree with the standards of the colors but from the UI standpoint I have to say using the primary color looks too flashy. I'd rather like it the way it looks in the first screenshots of the PR

Reply:
Will having a function that returns a suitable color for the question panel based on brightness (light/dark) work?

Reply:
@M123-dev Then please decide a correct primary color and a correct secondary color. That's what they're here for.

Reply:
Currently, I have hardcoded these colors based on the brightness of the app but I would like to receive feedback from you regarding this and change it accordingly if needed.

I have attached the screenshots of the results for this: 
`Theme.of(context).colorScheme.brightness == Brightness.dark ? const Color(0xff212121) : const Color(0xfff5f6fa)`

<div align="center">
<h3>Light mode</h3>
<img src="https://user-images.githubusercontent.com/63305824/159113069-6d0bdf73-ea2b-4d04-b5a6-8f940cff5067.png" width="250" height="400" />
</div>

<div align="center">
<h3>Dark mode</h3>
<img src="https://user-images.githubusercontent.com/63305824/159113068-0839eacc-0cf4-4c35-a638-f823cef0578c.png" width="250" height="400" />
</div>

Reply:
@GowthamGoush I think the color looks much better this way, we have several hardcoded values for colors in the app, but it would still probably be better to make them somehow centralized

Reply:
Looking at #1274, it looks like the color should be good by just removing the hardcoded value

Reply:
@GowthamGoush